tests-hivemq-platform-operator/src/integrationTest/java/com/hivemq/helmcharts/single/HelmCustomConfigIT.java

```diff
@@ -71,18 +71,28 @@ void withCustomXml_hivemqRunning() throws Exception {
     void withExistingConfigMap_customResourceCreated() throws Exception {
         installPlatformOperatorChartAndWaitToBeRunning();
         final var configMap = K8sUtil.createConfigMap(client, platformNamespace, "hivemq-config-map.yml");
+        final var configMapName = configMap.getMetadata().getName();
 
-        installPlatformChartAndWaitToBeRunning("--set",
-                "config.create=false",
-                "--set",
-                "config.name=" + configMap.getMetadata().getName());
+        installPlatformChartAndWaitToBeRunning("--set", "config.create=false", "--set", "config.name=" + configMapName);
 
         await().atMost(ONE_MINUTE).untilAsserted(() -> {
             final var hivemqCustomResource =
                     K8sUtil.getHiveMQPlatform(client, platformNamespace, PLATFORM_RELEASE_NAME).get();
             assertThat(hivemqCustomResource.getAdditionalProperties().get("spec")).isNotNull()
                     .asString()
-                    .containsIgnoringCase("configMapName=hivemq-configuration");
+                    .containsIgnoringCase("configMapName=" + configMapName);
+            final var statefulSet =
+                    client.apps().statefulSets().inNamespace(platformNamespace).withName(PLATFORM_RELEASE_NAME).get();
+            assertThat(statefulSet).isNotNull();
+            assertThat(K8sUtil.getHiveMQContainer(statefulSet.getSpec()).getVolumeMounts().stream()) //
+                    .anyMatch(volumeMount -> volumeMount.getName().equals("broker-configuration") &&
+                            volumeMount.getMountPath().equals("/opt/hivemq/conf-k8s/"));
+
+            final var volumes = statefulSet.getSpec().getTemplate().getSpec().getVolumes();
+            assertThat(volumes).isNotNull();
+            assertThat(volumes.stream()) //
+                    .anyMatch(volume -> volume.getName().equals("broker-configuration") &&
+                            volume.getConfigMap().getName().equals(configMapName));
```

Comment:
The `ListAssert` has a direct `anyMatch()` assertion, so there is no need to call `stream()` manually. With this the assertions can be written a bit more compact:
```suggestion
            assertThat(K8sUtil.getHiveMQContainer(statefulSet.getSpec()).getVolumeMounts()) //
                    .anyMatch(volumeMount -> volumeMount.getName().equals("broker-configuration") &&
                            volumeMount.getMountPath().equals("/opt/hivemq/conf-k8s/"));
            assertThat(statefulSet.getSpec().getTemplate().getSpec().getVolumes()).isNotNull()
                    .anyMatch(volume -> volume.getName().equals("broker-configuration") &&
                            volume.getConfigMap().getName().equals(configMapName));
```